File: backend/src/basico/002-ejercicios/004-valor_ascii/ascii.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-

if __name__ == "__main__":
    caracter = "a"
    caracter_en_ascii = ord(caracter)
    print("El caracter '%s' en ASCII ---> %d" % (caracter, caracter_en_ascii))

    numero = 89
    numero_en_caracter = chr(numero)
    print("El número %d en caracter ---> %s" % (numero, numero_en_caracter))

File: backend/src/basico/002-ejercicios/004-valor_ascii/ascii.rb
#!/usr/bin/ruby

if (__FILE__ == $0)
    caracter = "a"
    caracter_en_ascii = caracter.ord
    puts "El caracter '%s' en ASCII ---> %d" % [caracter, caracter_en_ascii]

    numero = 98
    numero_en_caracter = numero.chr
    puts "El número %d en caracter ---> %s" % [numero, numero_en_caracter]
end
